systemtest/src/test/resources/upgrade/StrimziDowngradeST.yaml

```diff
@@ -0,0 +1,49 @@
+# --- Structure ---
+#- fromVersion: String - From this version we will downgrade the CO
+#  toVersion: String - From this version we will upgrade the CO to latest = "HEAD"
+#  fromExamples: String - Folder name which contains examples to deploy. Downloaded from "urlFrom"
+#  toExamples: String - Folder name which contains examples to upgrade / downgrade to. Downloaded from "urlTo"
+#  urlFrom: String - url, from which examples are downloaded
+#  urlTo: String - url, from which examples are downloaded during downgrade
+#  additionalTopics: int - number of additional topics, which will be created
+#  oldestKafka: String - version of Kafka, used in initial deployment of Kafka CR
+#  imagesAfterOperations:  Strings - images that corresponding pods should contain. They are checked after upgrade
+#    zookeeper: 
+#    kafka: 
+#    topicOperator: 
+#    userOperator: 
+#  deployKafkaVersion: String - version of Kafka, used in deployment
+#  client:
+#    continuousClientsMessages: int - message count sent during upgrade
+#  environmentInfo:
+#    maxK8sVersion: String - maximal version of Kubernetes on which the upgrade test runs
+#    status: stable  String - info about stability of the testing environment
+#    flakyEnvVariable: String - name of the environmental variable which is flaky on current environment
+#    reason: String - reason why we skip / not skip the test case (information, not propagated to test itself)
+#  featureGatesBefore: String - FG added to `STRIMZI_FEATURE_GATES` environment variable, on initial deploy of CO
+#  featureGatesAfter: String - FG added to `STRIMZI_FEATURE_GATES` environment variable, on upgrade of CO
+# --- Structure ---
+  
+- fromVersion: HEAD
+  toVersion: 0.28.0
```

Comment:
Looks like we lost the 0.29.0 upgrade in this PR. I think the YAML needs to be regenerated.